ci/install_deps.sh

```diff
@@ -3,7 +3,13 @@
 set -e
 set -x
 
+# See https://github.com/pypa/pip/issues/7953
+echo "import site
+import sys
+site.ENABLE_USER_SITE = '--user' in sys.argv[1:]
+$(cat ./ibis/setup.py)" > ./ibis/setup.py
```

Comment:
Ideally this will be fixed in the ibis repo: https://github.com/ibis-project/ibis/issues/2738